src/api/RequestApi.ts

```diff
@@ -238,13 +238,13 @@ export type IInRequest = {
   /** Required - The Employee's Office */
   office: string;
   /** Required - Can only be 'true' if it is a New to USAF Civilain.  Must be 'false' if it is a 'mil' or 'ctr' */
-  isNewCivMil: "yes" | "no";
+  isNewCivMil: "yes" | "no" | "";
   /** Required - The user's previous organization.  Will be "" if isNewCiv is false */
   prevOrg: string;
   /** Required - Can only be 'true' if is a Civ/Mil.  For Ctr, will be 'false' */
```

Comment:
Update comment to
/** Required - Can only be 'true' if it is Civilian or Military.  Must be '' if it is a Contractor */